specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json

```diff
@@ -1129,6 +1132,14 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "Gets outbound rules that use this backend address pool."
         },
+        "outboundRules": {
```

Comment:
update a stable version is breaking change, pls make the change in new service tag.